spec/models/user_spec.rb

```diff
@@ -218,4 +218,260 @@
       end
     end
   end
+
+  describe 'paper_trail', versioning: true do
+    context 'creating user' do
+      context 'computacenter relevant' do
```

Comment:
The outer context is empty, so if you combined the two lines like `context 'creating a user relevant to computacenter' do` then you wouldn't need to raise the `RSpec/NestedGroups` limit?

Reply:
the context is empty but separates it from the other contexts and helps with grouping. i'm not too fussed about this kind of thing